packages/trpc/server/routers/viewer/organizations/update.handler.ts

```diff
@@ -32,17 +32,24 @@ const updateOrganizationSettings = async ({
   input: TUpdateInputSchema;
   tx: Parameters<Parameters<PrismaClient["$transaction"]>[0]>[0];
 }) => {
-  // if lockEventTypeCreation isn't given we don't do anything.
-  if (typeof input.lockEventTypeCreation === "undefined") {
-    return;
+  const data: Prisma.OrganizationSettingsUpdateInput = {};
+
+  if (input.hasOwnProperty("lockEventTypeCreation")) {
+    data.lockEventTypeCreationForUsers = input.lockEventTypeCreation;
+  }
+
+  if (input.hasOwnProperty("adminGetsNoSlotsNotification")) {
+    data.adminGetsNoSlotsNotification = input.adminGetsNoSlotsNotification;
   }
```

Comment:
We have to construct data property so we only update what has changed - allows us to pass in only the orgSetting we want to update and not have the previous state of for data before